tf_encrypted/convert/register.py

```diff
@@ -598,6 +601,37 @@ def _avgpool(converter, node: Any, inputs: List[str]) -> Any:
 
   return out
 
+def _global_avgpool(converter, node: Any, inputs: List[str]) -> Any:
```

Comment:
let's follow the trend of `_keras_{layer_identifier}` here

Reply:
@jvmancuso done